modules/Challenge.js

```diff
@@ -172,6 +172,25 @@ class Challenge {
         });
     }
 
+    /**
+    * Returns current state of database.
+    * @returns {Promise<any>}
+    */
+    static getCurrentDbState() {
+        return new Promise((resolve, reject) => {
+            SystemStorage.connect().then(() => {
+                SystemStorage.runSystemQuery('SELECT id, time, block_id, answer, dh_id, import_id, answered FROM data_challenges', [])
```

Comment:
lets try to use sequlize instead of custom raw system query

Reply:
https://github.com/OriginTrail/ot-node/blob/3683a24e1e6f551c3436ccca4f7126af749152dc/modules/RemoteControl.js#L30